add-schema.sh

```diff
@@ -38,7 +38,7 @@ fi
 # Add submodule
 if [ ! -d "schemas/${schema}" ]; then
   echo "Adding schema from ${gitRepository}, branch  ${gitBranch} to schemas/${schema}"
-  git submodule add -b ${gitBranch} ${gitRepository} schemas/${schema}
+  git submodule --force add -b ${gitBranch} ${gitRepository} schemas/${schema}
```

Comment:
`--force` must be after `add`:
```
git submodule add --force -b ${gitBranch} ${gitRepository} schemas/${schema}
```

Reply:
Committed the right order in https://github.com/geonetwork/core-geonetwork/commit/887af4d4e8cf6abc4c5b281fd30ebc4900e26ae5